### ISSUE TITLE
errors in `load`ed Tiltfiles are missing key info

### ISSUE THREAD

**Issue by landism** [windmilleng/tilt#2271]
repro:

1. In Tiltfile:
```
load('Tiltfile.inc')
```
2. In Tiltfile.inc:
```
local('asfasdfasdfasfd')
```

observed:
```
Traceback (most recent call last):
  /private/tmp/Tiltfile:1:1: in <toplevel>
Error: cannot load Tiltfile.inc: command ["sh" "-c" "asdf"] failed.
error: exit status 127
```

expected:
the output to include:
1. The Tiltfile.inc stack (or at least filename:lineno)
2. the fact that it was running `local`

**Comment by landism** [comment]
`include` gives us a stack from Tiltfile.inc, but no stack from Tiltfile (i.e., what line has the `include`?), which could be important in rare cases, but seems much lower priority:
```
Traceback (most recent call last):
  /private/tmp/Tiltfile.inc:2:6: in <toplevel>
  <builtin>: in local
Error: command ["sh" "-c" "asdf"] failed.
error: exit status 127
```

**Comment by landism** [comment]
this was fixed in #2443
